pkg/asset/installconfig/ovirt/validaton_test.go

```diff
@@ -83,3 +83,38 @@ func Test_validateAuth(t *testing.T) {
 func CreateMockOvirtServer(handler http.HandlerFunc) *httptest.Server {
 	return httptest.NewServer(handler)
 }
+
+func Test_validateURL(t *testing.T) {
+	tests := []struct {
+		url           string
+		expectSuccess bool
+	}{
+		{
+			url:           "engine.example.com",
+			expectSuccess: false,
+		},
+		{
+			url:           "ftp://engine.example.com",
+			expectSuccess: false,
+		},
+		{
+			url:           "http://engine.example.com",
+			expectSuccess: false,
+		},
+		{
+			url:           "https://engine.example.com",
+			expectSuccess: true,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.url, func(t *testing.T) {
+			// validateURLFunc := validateURL()
+			// got := validateURLFunc(tt.url)
```

Comment:
We should avoid merge commented lines which are not describing the function.

Reply:
Yeah you're right, I'll remove it